proposals/devcontainer-distribution.md

```diff
@@ -0,0 +1,71 @@
+# Community creation of templates and features and their distribution.
+
+This proposal is created to arrive at a schema where different communities and organizations can create features and devcontainer templates and make them discoverable to users.
+
+The proposal centers on the following ideas:
+
+- Discoverability for users both on the web (github.com) and on any tools that implement the [Dev container specification](../docs/specs/devcontainer-reference.md).
+- Users can validate the origin of the template or feature that they want to use.
+- Users can pin a particular version of a feature to allow consistent repeatable environments.
+
+# Proposal
+
+## Github Topics
+
+This can be achieved with the use of [GitHub Topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics), specifically the [devcontainers](https://github.com/topics/devcontainers) topic. Additionally the repositories can be marked with **devcontainers-templates** and/or **devcontainers-features** to mark specifically the contents of the repository.
+
+Any UI application can leverage this search to provide a list of repositories that contain features or templates and present them for their users. The application would query the latest release and obtain the details to show them to the users.
+
+An implementation of the spec can also use other mechanisms to speed up or improve the user experience while searching.
+
+At the same time it should contain a way for users to directly reference a repository containing the features/templates that they want to use.
+
+## Releases
+
+As a distribution and storage mechanism we propose using GitHub releases. The releases would contain compressed files with the source code of the feature or template. Together with the release the corresponding metadata file would be included either **devcontainer-feature.json**, **devcontainer-template.json** or **devcontainer-collection.json**.
+
+## Collections
+
+Features and/or templates can be distributed both as a single unit or as a **collection** of them. In case they are released as a collection the release should include a **devcontainer-collection.json** file that groups the information of each feature/template contained.
```

Comment:
```suggestion
Features and/or templates can be distributed both as a single unit or as a **collection** of them. In case they are released as a collection, the release should include a **devcontainer-collection.json** file that groups the information of each feature or template contained.
```